libs/blocks/merch/merch.js

```diff
@@ -828,6 +828,12 @@ export async function buildCta(el, params) {
       cta.setAttribute('aria-label', ariaLabel);
     });
   }
+
+  // @see https://jira.corp.adobe.com/browse/MWPW-173470
+  cta.onceSettled().then(() => {
+    if (getConfig()?.locale?.prefix === '/kr' && cta.value[0].offerType === OFFER_TYPE_TRIAL) cta.remove();
```

Comment:
I see optional chaining is used everywhere else in this method, it's probably safer you use it as well
```suggestion
    if (getConfig()?.locale?.prefix === '/kr' && cta.value[0]?.offerType === OFFER_TYPE_TRIAL) cta.remove();
```